lib/drive-get-experiment.js

```diff
@@ -0,0 +1,42 @@
+const Hyperdrive = require('hyperdrive')
+const Corestore = require('corestore')
+const Experiment = require('./experiment')
+
+class DriveGetExperiment extends Experiment {
+  constructor (tmpDir, { nrFiles, fileByteSize }) {
+    super(tmpDir, 'Drive Get Hyperdrive experiment')
+
+    this.store = new Corestore(this.tmpDir)
+    this.drive = new Hyperdrive(this.store)
+
+    this.nrFiles = nrFiles
+    this.fileByteSize = fileByteSize
+
+    this.file = 'a'.repeat(this.fileByteSize)
+  }
+
+  async _setup () {
+    if (this.closing) return
+    for (let i = 0; i < this.nrFiles; i += 1) {
+      if (this.closing) return
+      await this.drive.put(`/blob${i}.txt`, this.file)
+    }
+  }
+
+  async _runExperiment () {
+    for (let i = 0; i < this.nrFiles; i += 1) {
+      if (this.closing) return
+      const content = await this.drive.get(`/blob${i}.txt`)
+      if (content == null) {
+        throw new Error('Bug in this code: nullish entry in the hyperdrive')
+      }
+    }
```

Comment:
let's add another sanity check here, to make sure the experiment ran at least once (currently it doesn't, because there's a bug in the config parsing). Something like

```
    let ranAtLeastOnce = false

    for (let i = 0; i < this.nrFiles; i += 1) {
      if (this.closing) return
      const content = await this.drive.get(`/blob${i}.txt`)
      ranAtLeastOnce = true
      if (content == null) {
        throw new Error('Bug in this code: nullish entry in the hyperdrive')
      }
    }

    if (!ranAtLeastOnce) throw new Error('Bug in this code: no entry was fetched')
```

When running on the example config (`HYPERCORE_SCALE_TEST_INTERVAL_MS=5000 HYPERCORE_SCALE_EXPERIMENTS_FILE_LOC=example-config.json node run.js | pino-pretty`), this will now throw (with the original code it just reports experiments taking 0s)

Reply:
i've added this code to the experiment. Would it be useful to check if `nrFiles` is greater than 0 in the constructor?

Reply:
That's not needed, this sanity check would catch that